app/views/home/list_items.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="flex flex-col items-center my-4 w-full">
+  <h1 class="text-2xl font-bold mb-6">My Items</h1>
+
+  <% if @items.any? %>
+    <div class="w-full grid gap-4 grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-4 2xl:grid-cols-4">
+      <% @items.each do |item| %>
+        <div class="bg-white rounded-xl shadow p-4 text-center w-full aspect-square flex flex-col justify-center items-center">
+          <div class="mb-3">
+            <% if item.image_url.present? %>
+              <div class="mb-3">
+                <%= image_tag item.image_url,
+                    class: "w-60 h-60 mb-2 rounded-lg object-cover cursor-pointer hover:opacity-80 transition-opacity",
+                    onerror: "this.style.display='none'; this.nextElementSibling.style.display='block';",
+                    onclick: "openImageModal('#{item.image_url}', '')" %>
+                <div class="w-60 h-60 mb-2 rounded-lg bg-gray-200 flex items-center justify-center text-gray-500 text-xs cursor-pointer hover:opacity-80 transition-opacity"
+                     style="display: none;"
+                     onclick="openImageModal('#{item.user_items.first.image_url}', '#{item.name}')">
+                  Image Error
+                </div>
+              </div>
+            <% else %>
+              <p class="">?</p>
+            <% end %>
+          </div>
+        </div>
+      <% end %>
+    </div>
+  <% else %>
+    <div class="bg-white rounded-xl shadow p-8 text-center w-full max-w-md">
+      <p class="text-gray-600 text-lg">No items found</p>
+      <p class="text-gray-500 text-sm mt-2">You haven't collected any items yet.</p>
```

Comment:
It does not mean like this. It should be "There's no items in the DB" which will never happen when we launch the site :)
btw just leave it, we'll remove this entire screen later